spec/ChatroomSpec.js

```diff
@@ -47,6 +47,12 @@ describe('Chatroom', function() {
     done();
```

Comment:
Could we test for the presence of a message, rather than the internal details of a message. The functioning of the message component is tested seperately.